model/src/main/kotlin/com/testerum/model/runner/tree/id/RunnerIdCreator.kt

```diff
@@ -0,0 +1,63 @@
+package com.testerum.model.runner.tree.id
+
+import com.testerum.model.feature.hooks.HookPhase
+import com.testerum.model.infrastructure.path.Path
+import com.testerum.model.step.StepCall
+import com.testerum.model.test.TestModel
+
+object RunnerIdCreator {
+
+    fun getRootId(): String {
+        return "root"
+    }
+
+    fun getFeatureId(
+        parentNodeId: String,
+        nodePath: Path
+    ): String {
+        return "$parentNodeId/${nodePath.directories.last()}"
+    }
+
+    fun getTestId(
+        parentNodeId: String,
+        test: TestModel
+    ): String {
+        return "${getRootId()}/${test.id}"
```

Comment:
this looks like a bug: shouldn't we use the `parentNodeId` instead of the root id? If it's a bug, please fix the methods below also. If not, let's add comments explaining why we're ignoring the parent id.

Reply:
TestId already have the parentID in it. This is the reson I don't need the parent ID. Parent ID was removed as parameter